internal/npm_package/test/BUILD.bazel

```diff
@@ -1,10 +1,11 @@
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
```

Comment:
this is a breaking change for our users. They don't currently depend on the `@bazel_skylib` repository, and I don't want to introduce such a transitive dependency because Bazel's support for transitive dependencies is so broken - e.g. how will my users get a correct WORKSPACE file with correct version of skylib as we make changes?

I think we can vendor the `write_file.bzl` into our third_party directory like we do for `paths.bzl`

Reply:
Done.